lib/Zonemaster/Backend/DB.pm

```diff
@@ -196,8 +196,6 @@ sub _normalize_params {
     $normalized{ipv4}     = $$params{ipv4}      // $profile->get( 'net.ipv4' );
     $normalized{ipv6}     = $$params{ipv6}      // $profile->get( 'net.ipv6' );
     $normalized{profile}  = $$params{profile}   // "default";
-    $normalized{priority} = $$params{priority}  // 10;
-    $normalized{queue}    = $$params{queue}     // 0;
```

Comment:
Why are priority and queue removed from the normalization? That seems wrong to me. Maybe I'm missing something?

Reply:
They are removed because they shouldn't affect the fingerprint computation. And this normalization layer has been added to filter the parameters kept to compute the fingerprint.
We decided not to keep these parameters because they don't affect the outcome of a test. Having 2 tests with all parameters equal beside their priority (or queue) should not give a different fingerprint.

Reply:
I see, I added this method while working on the fingerprint. I now realize that its name might be ambivalent. This is not exactly a _normalization_ per se, but more a _projection_ of the parameters. This might indeed need some refactoring.